### PR TITLE
[Enhancement] Add tips to use jdk11

### PR DESCRIPTION
**Pull request by gengjun-git**
## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.1
  - [x] 3.0
  - [x] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: bin/start_fe.sh [hunk 1]
```diff
@@ -115,6 +115,10 @@ if [[ "$JAVA_VERSION" -gt 8 ]]; then
     final_java_opt=$JAVA_OPTS_FOR_JDK_9
 fi
 
+if [[ "$JAVA_VERSION" -lt 11 ]]; then
+    echo "tips: current jdk version is $JAVA_VERSION, using jdk 11 or higher will provide better performance"
```
Comment (imay) [inline bin/start_fe.sh:119]: Better to give more reason why higher JDK can provide better performance.

Comment (gengjun-git) [reply bin/start_fe.sh]: added
better GC performance.

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by Astralidea — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: bin/start_fe.sh [hunk 2]
```diff
@@ -115,6 +115,10 @@ if [[ "$JAVA_VERSION" -gt 8 ]]; then
     final_java_opt=$JAVA_OPTS_FOR_JDK_9
 fi
 
+if [[ "$JAVA_VERSION" -lt 11 ]]; then
+    echo "tips: current jdk version is $JAVA_VERSION, using jdk 11 or 17 will provide better GC performance"
```
Comment (nshangyiming) [inline bin/start_fe.sh:119]:
```suggestion
    echo "Tips: current JDK version is $JAVA_VERSION, JDK 11 or 17 is highly recommended for better performance(lower version JDK may not be supported in the future)"
```

Comment (gengjun-git) [reply bin/start_fe.sh]: done

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by nshangyiming — APPROVED** [review]

**Review by Astralidea — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.1

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Pull request merged**